### PR TITLE
Upgrade of the LiipImagineBundle.

### PR DESCRIPTION
**Pull request by szymach**
And also of Behat and it's components.

### CODE REVIEW

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

File: behat.yml [hunk 1]
```diff
@@ -1,16 +1,30 @@
...
+                    selenium2: ~
+        SensioLabs\Behat\PageObjectExtension:
+            namespaces:
+                page:
+                    - FSi\Bundle\GalleryBundle\Behat\Context\Page
+travis:
+    <<: *default
+    extensions:
+      Behat\MinkExtension:
+        base_url: http://gallery-bundle.dev:8080/app_test.php/
```
Comment (JarekW) [inline behat.yml]: `base_url` is the same as in `&default` so there is no need for this profile

**Comment by JarekW** [comment]
If php server would run like:
```
php -S 127.0.0.1:8080
```
then it will be no need to create doman alias

**Comment by szymach** [comment]
@fsi-open

**Comment by szymach** [comment]
@fsi-open

**Comment by bartekgruszka** [comment]
:+1:

**Comment by szymach** [comment]
@fsi-open

**Comment by JarekW** [comment]
:+1:

**Comment by szymach** [comment]
@fsi-open plis

**Comment by szymach** [comment]
@fsi-open

**Comment by chives** [comment]
:+1:

**Pull request merged**